examples/cpp/pyperf/PyPerfProfiler.cc

```diff
@@ -226,16 +226,23 @@ bool PyPerfProfiler::populatePidTable() {
   // Populate config for each Python Process
   auto pid_config_map = bpf_.get_hash_table<int, PidData>(kPidCfgTableName);
 
+  logInfo(3, "Pruning dead pids\n");
+  auto pid_config_keys = pid_config_map.get_keys_offline();
+  for (const auto pid : pid_config_keys) {
```

Comment:
So let's make sure I understand this logic: for each PID in the PIDs map, if it doesn't exist in the list of current PIDs, remove it. If it exists, remove it from the list of current PIDs, so we don't `tryTargetPid` them again?
The only "problem" I see here is if Python 3.6 execs Python 3.7 and then we end up using the wrong offsets. Don't care much about this case, and I assume we'll move to `do_exit`/`execve` probes anyway.

Reply:
(Or if there's a PID reuse within 1 second, less likely)

Reply:
Yes.